src/frontend/src/lib/components/ui/ImgBanner.svelte

```diff
@@ -11,6 +11,6 @@
 
 <div class="relative block" class:h-56={size === 'big'} class:h-40={size === 'small'}>
 	<div class="absolute h-full w-full">
-		<Img {src} {alt} {width} {height} {fitHeight} />
+		<Img {src} {alt} {width} {height} {fitHeight} styleClass="object-cover" />
```

Comment:
just to double confirm: this is the expected behaviors for all of the use cases of `ImgBanner`, correct? Info modal and Loader included

Reply:
yes also the images on the loader or the info modal should not be distorted.

Reply:
Yes, this is cleaner than a distorted image. We might later want to reduce the height of the image box when it gets squeezed, so its ratio stays in a reasonable range. but also then, cover is better than stretch.